### ISSUE TITLE
Pin definition does not change pin assignation to racing lines.

### ISSUE THREAD

**Issue by zhvial** [micbay/RaceTimerAndController#2]
In the code there is pin definition. It is confusing, because if you change the order of lines (lets say you wand A3 for lane1) it doesnt affect the program. 

const byte lane1Pin = PIN_A0;
const byte lane2Pin = PIN_A1;
const byte lane3Pin = PIN_A2;
const byte lane4Pin = PIN_A3;

All the "magic" is done in ISR routine where bits are checked. 

// Lane 1 positive trigger PINC = 0xXXXXXXX0
  // Lane 1 is on pin A0 which uses the 1st bit of the register; idx 0, of PINC Byte
  // Lane 2 positive trigger PINC = 0xXXXXXX0X
  // Lane 2 is on pin A1 which uses the 2nd bit of the register; idx 1, of PINC Byte
  // Lane 3 positive trigger PINC = 0xXXXXX0XX
  // Lane 3 is on pin A2 which uses the 3rd bit of the register; idx 2, of PINC Byte
  // Lane 4 positive trigger PINC = 0xXXXX0XXX
  // Lane 4 is on pin A3 which uses the 4th bit of the register; idx 3, of PINC Byte

I think that at least comment should be added that pins are not interchangeable

**Comment by micbay** [comment]
Yeah, this was originally the plan to make it settable, but wound up using a bit shifting approach to maximize the speed through the ISR() and that broke being able to use the lane#Pin variables for the interrupts which had the side effect of locking in the interrupt byte index value to the lane#.

I've refactored it now so that there is a lanes[][] array that sets the pin and interrupt byte pair for each lane #. So now if you want to wire the pins to different lanes you can do that by just changing the row its associated pin-byte pair is in. The row of lanes[] is equal to the lane. Row zero is reserved, row 1 is lane 1, etc.

The new lanes[] variable is described in code starting around line #52

I added an example commented in the new code at line 80, showing how one might now set pin A3 to be used by lane 1,

Let me know if this didn't cover the functionality you were looking for above.
